### DISCUSSION TITLE
Allow case insensitive regex

### DISCUSSION THREAD

**Discussion by MeikelLP** [grafana/grafana#48930] [Dashboards, Panels & Visualizations]
Overall I'd love to set regex flags in my "Filter data by values". If I want to filter for `wordpress` I have to write `[Ww]ord[Pp]ress` to make sure I catch everything...

This is very annoying (UX)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
